services/mailservice/mail.go

```diff
@@ -307,9 +309,14 @@ func sendMailUsingConfigAdvanced(mail mailData, config *model.Config, enableComp
 func SendMail(c smtpClient, mail mailData, fileBackend filesstore.FileBackend, date time.Time) *model.AppError {
 	mlog.Debug("sending mail", mlog.String("to", mail.smtpTo), mlog.String("subject", mail.subject))
 
-	htmlMessage := "\r\n<html><body>" + mail.htmlBody + "</body></html>"
+	htmlMessage := ""
+	if (mail.isBody) {
```

Comment:
This is a bit of glue code that is affecting everything else, can we just add the `<html><body>` tags directly to all the old templates, and leave the code as is? There is not reason after everything is migrated to have this differentiation. so I think is better to just leave the code as should be at the end.

Reply:
Great idea! Done